infrastructure/src/main/java/org/corfudb/infrastructure/datastore/DataStore.java

```diff
@@ -129,12 +133,19 @@ public synchronized void write(@Nonnull String key, @Nonnull Object value) {
                             String jsonPayload = JsonUtils.parser.toJson(value, value.getClass());
                             byte[] bytes = jsonPayload.getBytes();
```

Comment:
`String.getBytes` will create a new buffer, you can eliminate it by using the String instead.

Reply:
See below comments.